var/spack/repos/builtin/packages/llvm-openmp-ompt/package.py

```diff
@@ -33,26 +33,56 @@ class LlvmOpenmpOmpt(CMakePackage):
 
     homepage = "https://github.com/OpenMPToolsInterface/LLVM-openmp"
 
-    # tr4-stable branch
+    # towards_tr4 branch
+    version('towards_tr4', branch='towards_tr4',
+            git='https://github.com/OpenMPToolsInterface/LLVM-openmp.git')
+
     version('3.9.2b2',
             git='https://github.com/OpenMPToolsInterface/LLVM-openmp.git',
             commit='5cdca5dd3c0c336d42a335ca7cff622e270c9d47')
+
     # align-to-tr-rebased branch
     version('3.9.2b',
             git='https://github.com/OpenMPToolsInterface/LLVM-openmp.git',
             commit='982a08bcf3df9fb5afc04ac3bada47f19cc4e3d3')
 
+    # variant for building llvm-openmp-ompt as a stand alone library
+    variant('standalone', default=False,
+            description="Build llvm openmpi ompt library as a \
+                         stand alone entity.")
+
+    variant('build_type', default='Release',
+            description='CMake build type',
+            values=('Debug', 'Release', 'RelWithDebInfo', 'MinSizeRel'))
```

Comment:
i am sorry but there is a misunderstanding here: `build_type` is already implemented in `CMakePackage` from which `LlvmOpenmpOmpt` is derived. So you have this variant by default.

You only add it **again** (like what you have here) if you want to overwrite values. For example, in order to limit the values to  `Debug` and `Release` you add

```
variant('build_type', default='Release',
            description='CMake build type',
            values=('Debug', 'Release'))
```

If the default (`Debug', 'Release', 'RelWithDebInfo', 'MinSizeRel`) is OK, then there is no need to add it, just remove the variant altogether.